openghg_defs/data/site_info.json

```diff
@@ -267,10 +268,18 @@
     "ICOS": {
       "height": ["10m"],
       "height_name": ["10magl"],
-      "height_station_masl": 190.0,
-      "latitude": 58.39,
+      "height_station_masl": 219.0,
+      "latitude": 58.3886,
+      "long_name": "Birkenes Observatory, Norway",
+      "longitude": 8.2519
+    },
+    "ICOS": {
```

Comment:
Did we want to have two ICOS blocks here? Or could these be combined in some way?

Reply:
Nice spot, I have merged the two together